dask/distributed
# Set validate=True in all tests

- Fixed Worker validate parameter, which used to default to `distributed.scheduler.validate` instead of `distributed.worker.validate`
- `@gen_test` now accepts a `config` parameter
- Set validate=True when starting a Scheduler or Worker in all tests decorated by `@gen_test`
- Set validate=True when manually starting a Worker - or a subclass of Worker, such as BlockedGatherDep - in all tests decorated by `@gen_cluster`
- `@fail_hard` will no longer kill the whole test suite if a Worker which has been set to validate=False raises an exception
